docs/go-storage/operations/appender/commit_append.md

```diff
@@ -0,0 +1 @@
+`commit_append` is used to finish an append procedure.
```

Comment:
`commit_append` may bed added in a new proposal, please don't include it in this PR.

Reply:
Got it. And the appender support in go-storage-dropbox depends on this operation, do I need to remove `Appender` from the implementation section temporarily?

Reply:
Please wait my proposal and you don't need to remove it.